### DISCUSSION TITLE
Cannot init payment sheet (port error)

### DISCUSSION THREAD

**Discussion by doantrunghieu2607** [flutter-stripe/flutter_stripe#453] [Q&A]
I followed the guideline carefully, and I'm still not able to init payment sheet. Port from the simulation device changes every time I reset it. I change lib/config ip to my computer's ip address but it still does not work. How do I point the simulation device port to the one of lib/config?

**Comment by remonh87** [comment]
You can change it in https://github.com/flutter-stripe/flutter_stripe/blob/main/example/lib/config.dart
